lib/graphiti/adapters/jsonapi.rb

```diff
@@ -0,0 +1,89 @@
+module Graphiti
+  module Adapters
+    class JSONAPI < ::Graphiti::Adapters::Null
```

Comment:
This is probably more specifically a `GraphitiAPI` resource, as it assumes our request/response features, yes?

Reply:
Agreed, force-pushed